stable/yourls/values.yaml

```diff
@@ -0,0 +1,285 @@
+## YOURLS image version
+## ref: https://hub.docker.com/r/library/yourls/tags/
+##
+image:
+  registry: docker.io
+  repository: yourls
+  tag: 1.7.2
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistrKeySecretName
+
+## Site of the application
+## Used to construct short URLs
+## ref: https://hub.docker.com/_/yourls/
+##
+yourlsSite: http://yourls
+
+## User of the application
+## ref: https://hub.docker.com/_/yourls/
+##
+yourlsUsername: user
+
+## Application password
+## Defaults to a random 10-character alphanumeric string if not set
+## ref: https://hub.docker.com/_/yourls/
+##
+# yourlsPassword:
+
+## Table prefix
+## ref: https://hub.docker.com/_/yourls/
+##
+yourlsTablePrefix: yourls_
+
+## Set to `false` to allow the container to be started with blank passwords
+## ref: https://hub.docker.com/_/yourls/
+allowEmptyPassword: true
+
+# Extra environment variables which will be saved in a release-specific secret
+# or retrieved via valueFrom.
+# extraEnv:
+# - name: SECRET_TO_SAVE
+#   value: secret_value
+# - name: AWS_ACCESS_KEY_ID
+#   valueFrom:
+#     secretKeyRef:
+#       name: existing-secret
+#       key: access-key-id
+extraEnv: []
+
+replicaCount: 1
+
+externalDatabase:
+## All of these values are only used when mariadb.enabled is set to false
+  ## Database host
+  host: localhost:3306
+
+  ## non-root Username for YOURLS Database
+  user: yourls
+
+  ## Database password
+  password: ""
+
+  ## Database name
+  database: yourls
+
+##
+## MariaDB chart configuration
+##
+## https://github.com/helm/charts/blob/master/stable/mariadb/values.yaml
+##
+mariadb:
+  ## Whether to deploy a mariadb server to satisfy the applications database requirements. To use an external database set this to false and configure the externalDatabase parameters
+  enabled: true
+  ## Disable MariaDB replication
+  replication:
+    enabled: false
+
+  ## Create a database and a database user
+  ## ref: https://github.com/bitnami/bitnami-docker-mariadb/blob/master/README.md#creating-a-database-user-on-first-run
+  ##
+  db:
+    name: yourls
+    user: yourls
+    ## If the password is not specified, mariadb will generates a random password
+    ##
+    # password:
+
+  ## MariaDB admin password
+  ## ref: https://github.com/bitnami/bitnami-docker-mariadb/blob/master/README.md#setting-the-root-password-on-first-run
+  ##
+  # rootUser:
+  #   password:
+
+  ## Enable persistence using Persistent Volume Claims
```

Comment:
`mariadb.master.persistence` is the same as [the default values](https://github.com/helm/charts/blob/master/stable/mariadb/values.yaml#L133). It might be removed :slightly_smiling_face: